### ISSUE TITLE
Consider changing label for resource type "Service"

### ISSUE THREAD

**Issue by j0hj0h** [hbz/oerworldmap#1385]
Susan D'Antoni commented in context of the discussion about the landingpage #1086:

> I still have trouble with the term “services”.  And “repositories” might be a category, given their importance for searching.

As this has been a comment from other native speakers as well, we might consider renaming services. In #1340 (Define Subcategories for Services), repositories is already suggested as a subcategory. So the new word has to be broader. Could it be one of the following?

- OER Source
- Source
- Resource

**Comment by acka47** [comment]
In the [FAQ for editors](https://github.com/hbz/oerworldmap/wiki/FAQs-for-OER-World-Map-editors#service-organization-or-project) we write:

> A service is an online offer that provides OER-related contents or functionalities (e.g. it helps finding, assessing, creating or publishing OER). The most important subsection of services are OER repositories.

The only alternative name I can think of is the one we use in that description: "offer" or "online offer"

**Comment by j0hj0h** [comment]
As we have "Tools" now as a main category, we can move all offers which provide "functionalities" there and reduce it to offers providing "content" ... which gives the possibility to name it more specific.

**Comment by trugwaldsaenger** [comment]
I think things will become much clearer when we have the subcategories included, because everybody can see then, what kind of services exist. I think we should not change the structure of our datamodel in a hurry...

**Comment by acka47** [comment]
We should definitely keep in mind that we also have the "Tool" type when finalizing the subcategorization of "Service". Looking at #1340, at least one sub-category ("Authoring tools", maybe also "Learning platform") should rather be typed as "Tool" than as "Service". 

Anyway, what's the status of the sub-categories for "Service"? The PR at https://github.com/hbz/vocabs-edu/pull/26 is open for two years now...

**Comment by j0hj0h** [comment]
I don't say we should change anything in hurry, but discuss it again. I just picked it up, because @trugwaldsaenger posted Susan's comment in the landingpage google doc, and I think that it is a serious problem, that one who is not familiar with our datamodel yet, could quickly end up with the question "Where are the OER?", without finding an intuitive answer – just because of the wording.

**Comment by literarymachine** [comment]
@drrobertfarrow do you as a native speaker have any thoughts or suggestions on this?

**Comment by literarymachine** [comment]
Another thought that came up is not rename `Service`, but to add a new top-level type for anything content related (repositories, textbook platforms etc.). The could be [`Collection`](http://bib.schema.org/Collection) with a label of `OER Collections` or the like.

**Comment by trugwaldsaenger** [comment]
Here come my lessons learned from last weeks discussion:
1. Exchanging "Services" by "OER" or "OER`s" wont work. While it might be acceptable when searching for OER, it would be very irritating for the "[currently-on-the-map](https://oerworldmap.org/#on-the-map)"-view as well as for the "add-resource"-buttom.
1. Acceptable would be "OER-offer" or "OER-service". I like "OER-service", because I think it makes things clearer, while at the same time continues our wording so far.
1. Like @literarymachine pointed out, we could add another content related top-level type. 
1. I still believe, that things will become clearer, when we have our subcategories (#1340) in place.
1. As @j0hj0h pointed out, we might consider stressing second level types, whenever possible, e.g. speaking of "repositories" instead of "services", if the context allows to do so.
1. We should not mix up "services" with "tools". "Tools" basically includes the software used to develop/offer a service. If this is misleading as well, we might consider renaiming "tools", e.g. into "software" or similar.
1. One thing we did not discuss is the possiblity to include a direct link to the content-related second level types (repository, content-source, ...) as soon as we have implemented them. For example, we could include a link into the new slider of the landing page.
